### PR TITLE
fix-scopes

### PR DESCRIPTION
**Pull request by erickvneri**
Updated scopes syntax.  Fixed call of Weather API Key at lib\weather

This small update makes this Sample usable again so developers can access it as a useful example for their integrations.

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -102,7 +103,7 @@ function createConfigPage(pageId, currentConfig) {
             required: true,
             multiple: false,
             capabilities: ['colorControl', 'switch', 'switchLevel'],
-            permissions: ['r', 'x']
+            permissions: [ 'r:devices:*', 'x:devices:*' ]
```
Comment (erodewald) [inline lib/config.js]: This part was correct before – I'm not sure if you've got appropriate access, but take a look at [the spec](https://codesamsung.com/migration/iot/librarian/blob/master/swagger/smartapps.st.internal/v1.yml#L746-754) for a `DeviceSetting`. 

If you can't log in to the repo, here's the YAML spec:

```yaml
permissions:
          description: The required permissions for the selected device(s).
          type: array
          items:
            type: string
            enum:
            - r
            - x
            - w
```

File: lib/weather.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 const rp = require('request-promise');
 const prettyjson = require('prettyjson');
 const prettyjsonOptions = {};
+const colorType = require('../constants/colors');
+// Weather API config
+require('dotenv').config()
```
Comment (erodewald) [inline lib/weather.js]: A dotenv is a great idea, but could you please add an example with the expected keys? 

e.g.,: `.env.example`:
```bash
WEATHER_API_KEY=
```

Additionally, ensure that the `.env` is ignored by git, and `.env.example` is *not* ignored by git.

**Comment by erickvneri** [comment]
Hi Eric, I've committed my changes as you requested. Thank you for the spec reference. Let me know if it needs something else, please.

**Comment by erickvneri** [comment]
Thank you, guys!

**Pull request merged**